src/components/ChannelPreview/ChannelPreviewMessenger.js

```diff
@@ -1,11 +1,12 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import truncate from 'lodash/truncate';
 import styled from '@stream-io/styled-components';
 import PropTypes from 'prop-types';
 
 import { Avatar } from '../Avatar';
 import { themed } from '../../styles/theme';
-import { withTranslationContext } from '../../context';
+import { useChannelPreviewDisplayName } from './hooks/useChannelPreviewDisplayName';
+import useChannelPreviewDisplayAvatar from './hooks/useChannelPreviewDisplayAvatar';
```

Comment:
is there a reason these are exported/imported differently?

Reply:
changing the default export to a named one so we can ensure naming stays consistent throughout